pyproject.toml

```diff
@@ -33,6 +33,7 @@ dependencies = [
     "nerfacc==0.3.2",
     "open3d>=0.16.0",
     "opencv-python==4.6.0.66",
+    "Pillow==9.3.0",
```

Comment:
Does >=9.3.0 work?

Reply:
Sorry forgot about this, changed it to >= !!